### PR TITLE
packages: update to 3.0.0-beta.1

### LINKED ISSUE #58: Update components to 3.0.0-beta.1
This issue is raised in order to track the update of the following components.

Components that will be updated to 3.0.0-beta.1:
- runtime
- proxy

### PR DESCRIPTION
**Pull request by erick0z**
Update packages (cc-proxy, cc-runtime) to 3.0.0-beta.1.

Fixes #58

### CODE REVIEW

File: versions.txt [hunk 1]
```diff
@@ -14,15 +14,15 @@ qemu_lite_version=741f430a960b5b67745670e8270db91aeb083c5f
 # should contain the new versions.
 
 # Fedora
-proxy_obs_fedora_version=3.0.0alpha.3+git.b73e4a3-3.1
+proxy_obs_fedora_version=3.0.0beta.1+git.817d71e-1.1
 shim_obs_fedora_version=3.0.0alpha.4+git.ab14648-2.1
 image_obs_fedora_version=16670-31.1
```
Comment (gorozco1) [inline versions.txt]: I think we need to update also the image that these package depends on

**Review by gorozco1 — CHANGES_REQUESTED** [review]
We need to update the clear-containers-image these packages depends on..

File: versions.txt (hunk 1 above)
Comment (erick0z) [reply versions.txt]: right!

**Review by gorozco1 — APPROVED** [review]

**Comment by erick0z** [comment]
@gorozco1 Can we merge this?

**Pull request merged**